bootstrap/sql/migrations/native/1.4.1/mysql/postDataMigrationSQLScript.sql

```diff
@@ -0,0 +1,11 @@
+-- matchEnum Test Definition Parameter for columnValuesToBeInSet 
+UPDATE test_definition
+set json = JSON_MERGE_PRESERVE(
```

Comment:
hi sorry silly question, this does not belong to the use case defined in the PR right

Reply:
yes.. I branched off from a feature branch.. after the merge the diff should be cleaned